samples/android/app/src/main/res/values/strings.xml

```diff
@@ -15,7 +15,4 @@
         to attach a dictionary of values to the extras of the event ping. See MainActivity for
         where all of this is happening.
     </string>
-    <string name="send_ping_info">
```

Comment:
Same question as with glean-ac, can we remove the `send_baseline_ping` string as well as this one?